### DISCUSSION TITLE
is it framework agnostic or not ?

### DISCUSSION THREAD

**Discussion by rraammiinn** [dufu1991/stdf#19] [Q&A]
???

**Comment by dufu1991** [comment]
This is a component library based on Svelte and Tailwind CSS, meaning your project must use Svelte and configure Tailwind. You can create projects quickly using `pnpm create stdf`.

**Reply by rraammiinn** [reply]
thanks. 
I asked because it seems svelte components can be compiled to native web components which makes them framework agnostic .

**Reply by dufu1991** [reply]
If you have to, that's fine.
Svelte supports compiling a component as a frameless file, i.e. a separate JS file. However, compared to the component source code, the file size after compilation will be relatively large, if you only use a component, it is fine, if you want to use multiple components, it is very undesirable.
In addition, because STDF components are based on Tailwind, and components do not contain Tailwind CSS, so do not rely on Svelte to use, you must also ensure that the component to use the style

**Reply by rraammiinn** [reply]
ok , thanks .
